### PR TITLE
Initial draft for Sales Engineering

### PR DESCRIPTION
**Pull request by ryanrousseau**

### CODE REVIEW

**Comment by pstephenson02** [comment]
Just had a chance to read this and I think it looks great. There's a lot of stuff in there about being a good problem solver and teammate and everything which I love. I do think though that there could be more about being good advocates for our customers. Overall though 👍

File: Sales-Engineering.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+## Senior Solutions Architect
+
+> I am a well-rounded solutions architect and problem solver. I am trusted to autonomously lead people through complex problems with a proven record of delivering ideas from inception to successful completion. A Senior Solutions Architect encompasses all of the qualities listed in [Leadership](Leadership.md).
+>
+> I have extensive knowledge of Octopus Deploy and Continuous Delivery.
+
+### I am a technical mentor
+
+- People tend to ask my opinion when solving customer problems because I have the experience of solving various challenges and making wise choices..
```
Comment (peterlanoie) [inline Sales-Engineering.md:91]: extra period

Comment (peterlanoie) [reply Sales-Engineering.md]: fixed

**Review by pstephenson02 — APPROVED** [review]
:octocat: 🐙

**Review by peterlanoie — APPROVED** [review]

File: Sales-Engineering.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+> I am a highly sought-after member of an Octopus team with a proven record of providing world-class guidance to our customers.
+>
+> I have extensive knowledge of Continuous Delivery or extensive experience with Octopus Deploy.
+
+### I teach my team to help prospective and existing customers
+
+- I train teammates on Octopus features and how they impact customers.
+- I work with teammates on how to best showcase features or solve customer needs.
+- I pair with other members of the team to help them answer customer questions.
+- I help team members advocate on behalf of our customers to other Octopus teams.
```
Comment (octocrock) [inline Sales-Engineering.md:63]: I'm not sure of the right wording, but maybe also something like "I work with the customers to help them learn more about the best way to use Octopus and how different features can work for them."  And maybe "I work with the customer teams to assist in diversified use of Octopus" - for something about helping customers with getting Octopus usage up across different dev teams?  And different versions of those for the different levels of role.

File: Sales-Engineering.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+- Every venture has mundane tasks, and I work happily and diligently through them.
+- I go out of my way to develop a friendly and professional rapport with my customers and their team members.
+- My contributions to team discussions are net-positive, informative, and helpful.
+
+### I am learning how to help prospective and existing customers
+
+- I am learning about Octopus Deploy and its value proposition for our customers.
+- I am learning how to help customers discover if Octopus Deploy is the right tool for them.
+- I am learning how to help customers get the most out of their Octopus Deploy instance.
+- I pair with a senior member of the team to answer customer questions and provide guidance.
```
Comment (JChatmasOctopus) [inline Sales-Engineering.md:35]: Should this read "I pair with senior members..."?

File: Sales-Engineering.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+- I advocate on behalf of my customers within my team.
+
+## Senior Solutions Engineer
+
+> I am a valuable and enthusiastic member of an Octopus team. I show great potential and have a high growth trajectory toward independent contribution.
+>
+> I have a broad knowledge of Continuous Delivery or direct experience with Octopus Deploy.
```
[the remark this answers is not in the document]
Comment (JChatmasOctopus) [reply Sales-Engineering.md:42]: (referring specifically to the "or" in this sentence)

**Pull request merged**